src/pages/Home/components/movies-list/MoviesList.tsx

```diff
@@ -19,24 +24,52 @@ export const MoviesList: FC<IMoviesList> = ({moviesList}) => {
 
     const {updateMovieDetails} = useMovieDetailsContext()
 
+    useEffect(() => {
+        fetchMovies(filterBy, sortBy)
+    // eslint-disable-next-line
+    }, [])
+
+    // console.log(movies)
     return (
         <>
             <div className={styles.moviesCounter}>
                 <p>
-                    <b>39</b> movies found
+                    <b>{movies.length}</b> movies found
                 </p>
             </div>
-            <div className={classnames('row', styles.moviesList)}>{
-                moviesList.map((movieItem) => (
-                    <MovieCard
-                        key={movieItem.id}
-                        movieItem={movieItem}
-                        editMovie={showEditMovieModal}
-                        deleteMovie={showDeleteMovieModal}
-                        getMovieDetails={updateMovieDetails}
-                    />
-                ))
-            }</div>
+            {
+                movies.length ? (
+                    <div className={classnames('row', styles.moviesList)}>{
+                        movies.map((movieItem) => (
+                            <MovieCard
+                                key={movieItem.id}
+                                movieItem={movieItem}
+                                editMovie={showEditMovieModal}
+                                deleteMovie={showDeleteMovieModal}
+                                getMovieDetails={updateMovieDetails}
+                            />
+                        ))
+                    }</div>
+                ) : (
+                    !isLoading && <EmptyMovieList/>
+                )
+            }
+            {
+                isLoading && (
+                    <Loader />
+                )
+            }
+            {/*<div className={styles.loadMore}>
+                <Button
+                    buttonClassName={'addBtn'}
+                    buttonText={isLoading ? 'Loading...' : 'Load more'}
+                    buttonDisabled={isLoading}
+                    buttonAction={() => fetchMovies()}
+                />
+            </div>*/}
```

Comment:
Better remove unused code, you can find it git history in any time when you need it again.

Reply:
Removed